distributor-api-v1/operations.md

```diff
@@ -842,6 +843,7 @@ Gives a pricing information for the given configuration.
 |  | Property | Type | Description |
 | :--- | :--- | :--- | :--- |
 | `Client` | string | required | Identification of the client as described in [Authorization](https://mews-systems.gitbook.io/distributor-guide/distributor-api-v1/authorization). |
+| `ConfigurationId` | string | optional | GUID of the used Distributor configuration. If not specified, the configuration id which is set as property/hotel default will be used. |
```

Comment:
`Unique identifier of the Distributor configuration. If not...` We don't use GUID anywhere else in the docs. #consistency

Reply:
Oh, didn't know, I've reused the description already in the docs. So I'll rename all the usages of guid to unique identifier. https://github.com/MewsSystems/gitbook-distributor-guide/pull/22